app/routes/member/invites/$invitationId/index.tsx

```diff
@@ -0,0 +1,167 @@
+
+  return invitation;
+}
+
+export default function MemberInvitationIdRoute() {
+  const invitation = useLoaderData<typeof loader>();
+  const transition = useTransition();
+  const numberOfGuests = invitation.guests ? invitation.guests : 0;
+
+  const [isEditing, setIsEditing] = useState(false);
+  const [answer, setAnswer] = useState("RESPONDED_MAYBE");
+
+  function handleAnswerChange(e: React.ChangeEvent<HTMLInputElement>) {
+    setAnswer(e.target.value);
+  }
+
+  return (
+    <div className="mx-auto max-w-screen-md space-y-8 py-8">
+      <Link className="text-blue-600 underline" to="/dashboard">
+        Back
+      </Link>
+
+      <h2 className="text-lg font-bold">{invitation.event.schedule.title}</h2>
+
+      <span>
+        {format(parseISO(invitation.event.dateTime), "MMM dd 'at' h:mm b ")}{" "}
+        <span className="italic">
+          (
+          {formatDistance(parseISO(invitation.event.dateTime), new Date(), {
+            addSuffix: true,
+          })}
+          )
+        </span>
+      </span>
+
+      <section className="space-y-4">
+        {invitation.status === "SENT" && (
+          <div className="flex items-center gap-4 rounded border border-yellow-300 bg-yellow-100 p-4">
+            <FaExclamation className="text-yellow-600" />
+            You have not responded to this RSVP.
+          </div>
+        )}
+        {invitation.status === "RESPONDED_YES" && (
+          <div className="flex items-center gap-4 rounded border border-green-300 bg-green-100 p-4">
+            <FaCheck className="text-green-600" />
+            You have replied YES to this RSVP and are bringing{" "}
+            {invitation.guests} guests.
+          </div>
+        )}
+        {invitation.status === "RESPONDED_NO" && (
+          <div className="flex items-center gap-4 rounded border border-red-300 bg-red-100 p-4">
+            <ImCross className="text-red-600" />
+            You have replied NO to this RSVP.
+          </div>
+        )}
+        {invitation.status === "RESPONDED_MAYBE" && (
+          <div className="flex items-center gap-4 rounded border border-cyan-300 bg-cyan-100 p-4">
+            <FaQuestion className="text-cyan-600" />
+            You have replied MAYBE to this RSVP.
+          </div>
+        )}
+      </section>
+
+      <section className="space-y-4">
+        <h2 className="text-lg font-bold">Actions</h2>
+        <div className="flex items-center space-x-4">
+          {!isEditing && (
+            <button
+              className="rounded bg-green-600 py-2 px-4 font-bold text-white"
+              disabled={transition.state === "submitting"}
+              onClick={() => setIsEditing(true)}
+            >
+              Update RSVP
+            </button>
+          )}
+          {isEditing && (
+            <Form
+              action={`/member/${invitation.id}/update`}
+              method="post"
+              onSubmit={(e) => {
+                if (!confirm("Update Your RSVP?")) {
```

Comment:
Fully opinion territory here but I'm thinking it's not too important to have a confirmation step for this -- after all, they can just re-edit it if they made a mistake.